packages/serp-report/scripts/download.js

```diff
@@ -0,0 +1,26 @@
+import { writeFileSync } from 'fs';
+import { URL } from 'url';
+import { resolve } from 'path';
+import fetch from 'node-fetch';
+
+const DATA_URL = 'https://whotracks.me/data/trackers-preview.json';
+const OUTPUT_FILE = new URL(
+  '../src/background/trackers-preview.json',
+  import.meta.url,
+).pathname;
+
+const data = await fetch(DATA_URL).then(
+  (res) => (res.ok ? res.text() : ''),
+  () => '',
+);
+
+writeFileSync(
+  OUTPUT_FILE,
+  data || JSON.stringify({ trackers: {}, categories: [] }),
```

Comment:
Do we need the fallback here? I would expect the public URL from WhoTracksMe  `https://whotracks.me/data/trackers-preview.json` to be highly available. Only if AWS is down or if we mess up something in the update, it will fail.

In that (hopefully very unlikely) event, IMHO the script should fail, instead of succeeding with an empty data set.